### PR TITLE
Changes YogsMeta to have 4500 kPa air to distro

### PR DESCRIPTION
**Pull request by ToasterBiome**
#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md

# General Documentation

### Intent of your Pull Request

Brings YogsMeta further up to YogStation standards by having the distro be pressurized at 4500 kPa because it still had the "everything set to 101" pressure. Doesn't change ones that are not on at round start.

### Why is this change good for the game?

It isn't, YogsMeta sucks and I'm only helping change it so people dont cryo on round start.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Maybe the atmos wiki entry but it's already not updated

### What should players be aware of when it comes to the changes your PR is implementing?

Nobody sets the distro pressure anyway because they dont have to on YogsStation, so nothing will change except the pipes will actually be PRESSURIZED.

### What general grouping does this PR fall under? 
Atmos tweak

# Changelog

:cl:  
tweak: set active pump pressure to 4500 in yogsmeta atmos distro
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
at first i thought it was that you put 4500 kpa air in the pipes LMAAAOO

**Comment by Jarod1200** [comment]
yes i too enjoy breathing

**Comment by MegaEmpirical** [comment]
Yog station should be set to 101 not other stations brought up, someone is suposed to set it up.

**Comment by ToasterBiome** [comment]
> Yog station should be set to 101 not other stations brought up, someone is suposed to set it up.

Gonna have to hard disagree here. Not everything has to waste your time roundstart. Once YogStation was set to 4500 we are inescapably on a path to have the other maps follow suit. Other departments have "non optimal" round starts so they can make improvements over the shift, atmos round start would be annoying clicking while people yell at you to set 4500 or nobody does it at all until it's hindering them.

**Comment by MegaEmpirical** [comment]
It's very escapable, just depends on what Wej decides. I've never seen anyone yell about setting it up to 4500, I do see people complain about power which also needs to be set up.

**Review by patpol4 — CHANGES_REQUESTED** [review]
Having distro maxed absolutely makes sense, we don't want crew to suffocate if there's no atmos tech. 
You also changed waste pumps though, that isn't as essential, revert those to be in line with box.

**Comment by MegaEmpirical** [comment]
You also don't want power to go out. The station isn't supposed or he set up perfectly or even completely. Monster made this change with monstermos but it's really not that needed for it

**Comment by Jarod1200** [comment]
External airlocks that cycle do not work AT ALL if there is no air in distro. All maps should be made to have 4500 kpa on the distro pump if they don't have it already

**Comment by MegaEmpirical** [comment]
Fuck, I forgot about that. That's on me and that's fair. I concede.

**Review by patpol4 — CHANGES_REQUESTED** [review]
You're supposed to delete the var alltogether so it automatically sets it to the default, manually setting the default bloats the map file and vars

**Comment by ToasterBiome** [comment]
> You're supposed to delete the var alltogether so it automatically sets it to the default, manually setting the default bloats the map file and vars

Whoops, fixed. Thanks for telling me

**Review by patpol4 — APPROVED** [review]
All good
